frontend/test/setup.js

```diff
@@ -4,8 +4,6 @@ import Enzyme from 'enzyme'
 import chaiEnzyme from 'chai-enzyme'
 import Adapter from 'enzyme-adapter-react-16'
 import sinon from 'sinon'
-import EventSource from 'eventsourcemock'
-import AbortController from 'abort-controller'
```

Comment:
I think these dependencies should be removed from forntend/package.json

Reply:
Done, also moved them to dev dependencies as they are used only in tests.